.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
IMO, the repo shouldn't include IDE configuration files. I'd delete this file and add .vscode to the gitignore